aggsender/flows/flow_base.go

```diff
@@ -333,15 +342,57 @@ func (f *baseFlow) getBridgeExits(bridges []bridgesync.Bridge) []*agglayertypes.
 
 // getImportedBridgeExits converts claims to agglayertypes.ImportedBridgeExit objects and calculates necessary proofs
 func (f *baseFlow) getImportedBridgeExits(
-	ctx context.Context, claims []bridgesync.Claim,
+	ctx context.Context,
+	claims []bridgesync.Claim,
+	unclaims []bridgesynctypes.Unclaim,
 	rootFromWhichToProve common.Hash,
 ) ([]*agglayertypes.ImportedBridgeExit, error) {
+
+	// Build unclaim counts by GlobalIndex
+	unclaimCnt := make(map[string]int)
```

Comment:
I see that you are preserving the original ordering of the claims list when filtering unclaims, but I just wanted to understand better, so correct me if I am wrong:
- We can have a claim with global exit = 1, then unclaim global exit = 1, and then again correct claim with global exit = 1. Am I right when I say that this code, will remove the first claim with global exit = 1, and will leave the second claim with global exit = 1?